coralproject/talk
# [CORL-1557] Closed Stories Fix

## What does this PR do?

When a story is closed, this fixes the messaging around the form to improve the readability.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



With two different browsing contexts, go to the same stream. In one, close the story for commenting. In the other, try to post in the form, see the messaging around closed stories. Refresh either page, and see the more compact closed story UI!
